audit-implementation.md

```diff
@@ -36,6 +36,14 @@ class MongoAudit extends Model implements \OwenIt\Auditing\Contracts\Audit
         'new_values'   => 'json',
         'auditable_id' => 'integer',
     ];
+    
+    /**
+     * {@inheritdoc}
+     */
+    public function getTable(): string
+    {
+        return parent::getTable();
```

Comment:
The `getTable()` method in the `Audit` trait returns a `string`. Do you get any errors if you don't override the method?

Reply:
No, sorry that my mistake, PHPShtrom highlighted it as an error so I fixed it.